src/game.cpp

```diff
@@ -4214,7 +4224,7 @@ void Game::updateFrame(ProfilerGraph *graph, RunStats *stats,
 	if (draw_control->range_all) {
 		runData->fog_range = 100000 * BS;
 	} else {
-		runData->fog_range = 0.9 * draw_control->wanted_range * BS;
+		runData->fog_range = draw_control->wanted_range * BS;
```

Comment:
this change move the fog further out rather than at 90% (as discussed in other issues) - I don't see how this is related to this PR